### DISCUSSION TITLE
/posts/2023/06/18vitepress%E5%8D%9A%E5%AE%A2%E9%87%8C%E5%A2%9E%E5%8A%A0%E4%B8%80%E4%B8%AARSS%E8%AE%A2%E9%98%85

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#95] [Announcements]
# /posts/2023/06/18vitepress%E5%8D%9A%E5%AE%A2%E9%87%8C%E5%A2%9E%E5%8A%A0%E4%B8%80%E4%B8%AARSS%E8%AE%A2%E9%98%85

坚持深耕技术领域的T型前端程序员, 关注独立开发与AI，喜欢Vuejs、Nestjs, 还会点Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/posts/2023/06/18vitepress%E5%8D%9A%E5%AE%A2%E9%87%8C%E5%A2%9E%E5%8A%A0%E4%B8%80%E4%B8%AARSS%E8%AE%A2%E9%98%85

**Comment by Flow-sandyu** [comment]
感谢分享，我现在也搞好 rss 了 🎉
https://flow-sandyu.github.io/
